tavla/app/(admin)/components/IllustratedInfo/index.tsx

```diff
@@ -1,19 +1,30 @@
 import Image from 'next/image'
 import animals from 'assets/illustrations/Animals.png'
 import { Heading2, LeadParagraph } from '@entur/typography'
+import { PrimaryButton } from '@entur/button'
+import Link from 'next/link'
+import { CreateBoard } from '../CreateBoard'
+import { AddIcon } from '@entur/icons'
 
 function IllustratedInfo({
     title,
     description,
+    hasCreateButton = false,
```

Comment:
gjør det sånn her fordi vi også senere skal legge til en opprett organisasjon-knapp på tom org-side (bruker samme komponent)

Reply:
Smart tenkt! Men kan det heller være bedre å sende inn selve knappen i stedet? Hvis det skal legges opp for gjenbruk, så vil vel både teksten i knappen og href-en blant annet være forskjellig basert på om det er ny tavla eller ny org som skal lages

Reply:
Lurt! Prøvde å fikse det nå, sjekk gjerne om det ble sånn som du mente!